src/components/render-objects.vue

```diff
@@ -54,7 +54,8 @@ export default defineComponent({
     objects: {
       type: Array,
       required: false
-    }
+    },
+    class: String,
```

Comment:
what's this about?

Reply:
new PR for that: https://github.com/TeamFranka/affinity/pull/194